cdap-docs/developers-manual/source/advanced/application-logback.rst

```diff
@@ -8,19 +8,23 @@
 Application Logback
 ===================
 
-.. highlight:: xml
-
 YARN containers launched by a CDAP application use a default container logback file
 |---| ``logback-container.xml`` |---| packaged with CDAP and installed in 
 the CDAP :ref:`configuration directory <admin-manual-cdap-components>`. This logback does
 log rotation once every day at midnight and deletes logs older than 14 days. Depending on
-the use case, the default configuration may be sufficient.
+the use case, the default configuration may be sufficient. As long as the container is
+running, these policies will apply. (As the lifetime of many containers is often less than
+14 days, these limits may never be reached.)
+
+**Note:** In the case of the CDAP Standalone, the logback file used is ``logback.xml``, located
+in the ``<cdap-sdk-home>/conf`` directory.
 
-However, you can specify a custom ``logback.xml`` for a CDAP application by packaging
+You can specify a custom ``logback.xml`` for a CDAP application by packaging
```

Comment:
i checked with @yaojiefeng and @poornachandra, it has to be `logback.xml` for logback to pick it up.